R/pkgdown.R

```diff
@@ -3,6 +3,12 @@
 #' Opens the documentation website in a browser.
 #'
 #' @keywords internal
+#'
+#' @return Opens the {dm}-website as a side-effect
```

Comment:
Wording, here and everywhere:

```suggestion
#' @return Opens the {dm}-website as a side effect
```